controller/controller_server/utils.py

```diff
@@ -158,6 +158,24 @@ def validate_delete_snapshot_request(request):
     logger.debug("request validation finished.")
 
 
+def validate_expand_volume_request(request):
+    logger.debug("validating expand volume request")
+
+    if not request.volume_id:
+        raise ValidationException(messages.id_should_not_be_empty_message)
+
+    logger.debug("validating volume capacity")
+    if request.capacity_range:
+        if request.capacity_range.required_bytes < 0:
+            raise ValidationException(messages.size_bigger_than_0_message)
```

Comment:
I know it is an existing message, but now that I think about it, size **can** be 0, so it should be changed to something like `size can/should not be negative/lower than 0`

Reply:
done

Reply:
regarding the variable name, it is a good opportunity to align it with the others to not sound "ok".
i.e. `size_not_negative_message` sounds like the size was found to be not negative/required to be negative.
consider something like `size_should_not_be_negative_message`

Reply:
done